koap/src/jsMain/kotlin/adapter/messageToAdapted.kt

```diff
@@ -0,0 +1,117 @@
+package com.juul.koap.adapter
```

Comment:
I'd personally rename this file to `Message.kt`.
It's adapters for `Message` objects, so being in the `adapter` package with a filename of `Message.kt` seems pretty descriptive/organized to me. 🤷